src/modules/counter.test.js

```diff
@@ -0,0 +1,22 @@
+import { mealCounter } from './counter.js';
+
+describe('mealCounter', () => {
+  beforeEach(() => {
+    document.body.innerHTML = `
+        <div class="meal-count"></div>
+        <div class="container">
+            <div class="card"></div>
+            <div class="card"></div>
+            <div class="card"></div>
+            <div class="card"></div>
+            <div class="card"></div>
+            <div class="card"></div>
+        </div>`;
+    mealCounter();
+  });
+  test('should display the number of meals available', () => {
+    expect(document.querySelector('.meal-count').innerHTML).toBe(
+      '(6 recipes available)',
```

Comment:
Same here I think it is a good idea to test how the counter works if you have 0 meals. :eyes: :eyes: